msal/build.gradle

```diff
@@ -277,7 +277,7 @@ publishing {
 }
 
 tasks.whenTaskAdded { task ->
-    if (task.name == 'assembleDebug' || task.name == 'assembleRelease') {
+    if (task.name.contains('assemble') && !task.name.contains('Snapshot') && !task.name.contains('Test')) {
```

Comment:
I think it's better to have two separate if loops

`if (task.name.contains('assemble')) {
                task.dependsOn 'checkstyle', 'pmd', 'lint'
    }`

`if (task.name.contains('assemble') && !task.name.contains('Snapshot') && !task.name.contains('Test')) {
        task.dependsOn  'javadocJar', 'sourcesJar'
    }	    }`

Reply:
653bf3835853ae48449895940f772c68164a4856